code/src/components/NewThought.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+
```

Comment:
Check indentation for this component, I think you might have your tab set to 4 spaces instead of 2. 
Installing a formatter could be of help, I'm using Prettier sometimes ✨